src/main/java/de/numcodex/feasibility_gui_backend/query/broker/direct/FhirConnector.java

```diff
@@ -0,0 +1,146 @@
+package de.numcodex.feasibility_gui_backend.query.broker.direct;
+
+import static de.numcodex.feasibility_gui_backend.query.QueryMediaType.CQL;
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.hl7.fhir.r4.model.Bundle.BundleType.TRANSACTION;
+import static org.hl7.fhir.r4.model.Bundle.HTTPVerb.POST;
+
+import ca.uhn.fhir.context.FhirContext;
+import ca.uhn.fhir.rest.client.api.IGenericClient;
+import ca.uhn.fhir.rest.param.DateParam;
+import ca.uhn.fhir.rest.param.StringParam;
+import ca.uhn.fhir.rest.server.exceptions.BaseServerResponseException;
+import java.io.IOException;
+import java.io.InputStream;
+import org.hl7.fhir.instance.model.api.IBaseResource;
+import org.hl7.fhir.r4.model.Bundle;
+import org.hl7.fhir.r4.model.Library;
+import org.hl7.fhir.r4.model.Measure;
+import org.hl7.fhir.r4.model.MeasureReport;
+import org.hl7.fhir.r4.model.Parameters;
+
+public class FhirConnector {
+
+  private final FhirContext context;
+
+  private final IGenericClient client;
+
+  public FhirConnector(FhirContext context, IGenericClient client) {
+    this.context = context;
+    this.client = client;
+  }
+
+  /**
+   * Submit a {@link Bundle} to the FHIR server.
+   * @param bundle the {@link Bundle} to submit
+   * @throws IOException if the communication with the FHIR server fails due to any client or server error
+   */
+  public void transmitBundle(Bundle bundle) throws IOException {
+    try {
+      client.transaction().withBundle(bundle).execute();
+    } catch (BaseServerResponseException e) {
+      throw new IOException("An error occurred while trying to create measure and library", e);
+    }
+  }
+
+  /**
+   * Get the {@link MeasureReport} for a previously transmitted {@link Measure}
```

Comment:
I would say: "Evaluate a previously transmitted {@link Measure}." It's more concrete as simply "getting" and the method name says it also.